examples/audit/audit_examples/log_bulk_async_with_vault.py

```diff
@@ -0,0 +1,39 @@
+import os
+import time
+
+import pangea.exceptions as pe
+from pangea.config import PangeaConfig
+from pangea.services import Vault
+from pangea.services import Audit
+from pangea.services.audit.models import Event
+from pangea.tools import logger_set_pangea_config
+
+vault_token = os.getenv("PANGEA_VAULT_TOKEN")
+audit_token_vault_id = os.getenv("PANGEA_AUDIT_TOKEN_VAULT_ID")
+domain = os.getenv("PANGEA_DOMAIN")
+config = PangeaConfig(domain=domain)
+logger_set_pangea_config(logger_name="audit")
+
+# This example shows how to perform an audit log
+
+
+def main():
+    print("Logging bulk...")
+    vault = Vault(vault_token, config=config, logger_name="vault")
+    vault_response = vault.get(audit_token_vault_id)
+    audit_token = vault_response.raw_result["current_version"]["secret"]
```

Comment:
It would be nice to use result field as `vault_response.result.current_version.secret`